files/es/web/api/window/print/index.md

```diff
@@ -9,6 +9,7 @@ tags:
   - Referencia
   - Window
   - metodo
+browser-compat: api.Window.print
```

Comment:
@mfuji09 So these are not needed then?

Reply:
IMO, we can just remove them, the BCD and specification macros are still working.

See: https://github.com/mdn/yari/blob/5d8d88976164b6f653c3f2de688f3cd76f8537bb/kumascript/index.js#L56-L69

The parent document's metadata will be merged into the reflected translated document's metadata.

Reply:
Yup, I noticed that they _do_ work without these entries. I will clean it up.

Reply:
Removed.